static/src/stylesheets/layout/new-header/_veggie-burger.scss

```diff
@@ -58,6 +58,25 @@
     }
 }
 
+.veggie-burger__label {
+    @include fs-textSans(5);
+    bottom: -25px;
+    color: $neutral-3;
+    display: none;
```

Comment:
Ahaa, thanks. I added this to my list of question, because I couldn't find the mixin. Great!